### ISSUE TITLE
when I lein uberjar this project.It hangs on Compiling de.sveri.structconverter.db

### ISSUE THREAD

**Issue by EricShaoCN** [sveri/ffdc#1]
I run (start-all) , the project works fine.But when I lein uberjar this project,It hangs on Compiling de.sveri.structconverter.db.

eric@eric.com-Rev-1-0:~/cljprojects/testffdc/ffdc$ lein uberjar
Deleting files generated by lein-cljsbuild.
WARNING!!! version ranges found for:
[com.datomic/datomic-pro "0.9.5078"] -> [com.amazonaws/aws-java-sdk "1.8.11" :exclusions [javax.mail/mail org.apache.httpcomponents/httpclient commons-logging]] -> [com.amazonaws/aws-java-sdk-core "1.8.11"] -> [joda-time "[2.2,)"]
Consider using [com.datomic/datomic-pro "0.9.5078" :exclusions [joda-time]].

Compiling de.sveri.structconverter.service.friend
Compiling de.sveri.structconverter.globals
Compiling de.sveri.structconverter.server
Compiling de.sveri.structconverter.dev
Compiling de.sveri.structconverter.routes.home
Compiling de.sveri.structconverter.routes.base
Compiling de.sveri.structconverter.routes.csv
Compiling de.sveri.structconverter.db

**Comment by sveri** [comment]
Sorry, somehow I dont receive notifications for my own issues -.-
I look into this now.

**Comment by sveri** [comment]
Well, the problem is this line:
(let [db-conn (d/connect glob/db-uri)]
in server.clj.

It tries to connect to the database during compile time. In another project, where I used a similar approach I avoided it by using the hara component system (similar to stuarts), it did work there, if you implement the components in the correct way.
Another solution would be to defer connection with one of the possibilities clojure provides.
I will not update the repo in the next days, but I guess you get the idea?
